content/pt-pt/docs/setup/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+reviewers:
+- brendandburns
+- erictune
+- mikedanese
```

Comment:
```suggestion
```
Only include reviewers and approvers if you are confident they speak (European) Portuguese